package.json

```diff
@@ -41,6 +40,7 @@
     "@types/react-timeago": "^4.1.3",
     "autoprefixer": "^10.4.13",
     "postcss": "^8.4.21",
+    "prisma": "^5.9.0",
```

Comment:
can you revert it to the original version 4.11.0, just for consistency?

Reply:
oh yeah I thought I changed that I guess I didn't commit it